rebuild-backend.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+DIRECTORY=~/dynamodb-janusgraph-storage-backend
+
+rm -r ${DIRECTORY}/target
+rm -rf /tmp/searchindex
+rm -rf elasticsearch
+rm -r ${DIRECTORY}/server
+
+cd /${DIRECTORY}
+
+sudo src/test/resources/install-gremlin-server.sh
+
+sudo mvn install -X
```

Comment:
should not need sudo.

Honestly, probably better to remove rebuild-backend.sh from this PR and just keep it in your local repository. If you do not git add it, it will not appear in the commit.